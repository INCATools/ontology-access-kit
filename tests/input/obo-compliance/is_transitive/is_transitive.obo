format-version: 1.4
ontology: is_transitive

[Typedef]
id: R:1
is_transitive: true

