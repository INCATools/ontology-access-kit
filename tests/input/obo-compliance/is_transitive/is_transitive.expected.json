{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/is_transitive.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    } ]
  } ]
}